lib/model/frames/form.js

```diff
@@ -50,6 +50,7 @@ const AllVersions = Symbol('all versions');
 class Form extends Frame.define(
   table('forms'),
   'id',                                 'projectId',    readable,
+  'description',  readable, writable,
```

Comment:
Was it only forms and not projects that were an issue for the migration tests? So maybe those test failures will go away? (Though definitely a tricky set of questions there. 🤔)

Reply:
No, it won't fix the problem with the tests. The problem is that the code (`frames.js`) reflects the latest structure of the database (with `project.description` column), but the migration test starts by applying migrations only up to an earlier database state (without that description column). So the code to populate the database with the initial project doesn't work because it's expecting a database column that doesn't exist.

Reply:
Oh! That makes sense. Wow, that really is tricky.